model_analyzer/analyzer/perf_analyzer/perf_analyzer.py

```diff
@@ -48,75 +48,64 @@ def __init__(self, config):
         self._config = config
         self._outputs = []
 
-    def run_job(self, sweep_params):
+    def run(self):
         """
-        Parses the parameters to sweep over and runs
-        the perf_analyzer once per configuration.
-
-        Parameters
-        ----------
-        sweep_params : dictionary of lists
-            keys are arguments to perf_analyzer, values are
-            list of argument values to run perf_analyzer with.
+        Runs the perf analyzer with the
+        intialized configuration
 
         Returns
         -------
-        List of tuples
-            (params, output) where params are the set of
-            parameters provided for that run and output is
-            the PerfRecord for the perf_analyzer run
-
-        Raises
-        ------
-        TritonModelAnalyzerException
-            If _run_perf_analyzer throws this exception
-        """
-        # Create a config for each combination of parameters
-        combos = list(product(*tuple(sweep_params.values())))
-        run_configs = [dict(zip(sweep_params.keys(), vals)) for vals in combos]
-
-        # Collect a tuple (run_config, output)
-        run_outputs = []
-
-        # Run parameters are some subset of perf_analyzer config
-        for run_config in run_configs:
-
-            # update perf_analyzer config
-            for key, val in run_config.items():
-                self._config[key] = val
-
-            # Run perf_analyzer
-            run_record = self._run_perf_analyzer()
-            run_outputs.append((run_config, run_record))
-
-        return run_outputs
-
-    def _run_perf_analyzer(self):
-        """
-        Runs the perf_analyzer tool locally
-        and synchronously
-
-        Returns
-        -------
-        PerfRecord
-            output from the subprocess call to
-            the perf_analyzer
+        List of Records for metrics
+        that are generated by the perf analyzer
 
         Raises
         ------
         TritonModelAnalyzerException
             If subprocess throws CalledProcessError
         """
+
```

Comment:
Your formatter is not consistent. Empty line after comment here, but not in others.

Reply:
Yeah it leaves out some things, like empty lines between code blocks (which I assume is what it thought this was).